### ISSUE TITLE
Router S2_VPN script broken

### ISSUE THREAD

**Issue by fermosan** [apache/cloudstack#10073]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
VR
~~~

##### CLOUDSTACK VERSION

~~~
4.20
~~~

##### SUMMARY

When the Split Connection is enabled in the VPN Customer gateway the provision of the ipsec fails. 

##### STEPS TO REPRODUCE

~~~
Create a VPN Gateway for a Network
Create a VPN Customer Gateway wieh Split connections enable. 
Create a VPN connection with the VPN Customer gateway. 
Attempt to connect
~~~

##### EXPECTED RESULTS

~~~
VPN Connection should be established (Used to work before update to 4.20)
~~~

##### ACTUAL RESULTS

~~~
The IPSEC is missing from the virtual router.
Cloudstack agent reports:
2024-12-10 01:11:27,063 WARN  [kvm.resource.LibvirtComputingResource] (agentRequest-Handler-5:[]) (logid:9702fbd2) Execution of process [3888967] for command [/usr/share/cloudstack-common/scripts/network/domr/router_proxy.sh update_config.py 169.254.102.57 site_2_site_vpn.json.760285ff-2807-4e62-8b65-b66ac0b5d113 ] failed.
2024-12-10 01:11:27,063 WARN  [kvm.resource.LibvirtComputingResource] (agentRequest-Handler-5:[]) (logid:9702fbd2) Exception [Stream closed] occurred when attempting to run command [/usr/share/cloudstack-common/scripts/network/domr/router_proxy.sh update_config.py 169.254.102.57 site_2_site_vpn.json.760285ff-2807-4e62-8b65-b66ac0b5d113 ]. java.io.IOException: Stream closed

Error from running the provision script:

Traceback (most recent call last):
  File "/opt/cloud/bin/update_config.py", line 147, in <module>
    process_file()
  File "/opt/cloud/bin/update_config.py", line 57, in process_file
    finish_config()
  File "/opt/cloud/bin/update_config.py", line 42, in finish_config
    returncode = configure.main(sys.argv)
                 ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/cloud/bin/configure.py", line 1670, in main
    execIptables(config)
  File "/opt/cloud/bin/configure.py", line 1660, in execIptables
    iptables_executor.process()
  File "/opt/cloud/bin/configure.py", line 1566, in process
    vpns.process()
  File "/opt/cloud/bin/configure.py", line 1030, in process
    self.configure_ipsec(self.dbag[vpn])
  File "/opt/cloud/bin/configure.py", line 1103, in configure_ipsec
    if splitconnections and peerlistarr.count > 1:
                            ^^^^^^^^^^^^^^^^^^^^^
TypeError: '>' not supported between instances of 'builtin_function_or_method' and 'int'

~~~

**Comment by weizhouapache** [comment]
The fix: #10067

**Comment by weizhouapache** [comment]
fixed by #10067
